tests/lib/mkpinentry.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 
-echo "setup fake gpg pinentry environment"
-mkdir -p ~/.snap/gnupg/
-echo pinentry-program "$TESTSLIB/pinentry-fake.sh" > ~/.snap/gnupg/gpg-agent.conf
-chmod -R go-rwx ~/.snap
+if [ gpg --version | MATCH "gpg (GnuPG) 1." ]; then
```

Comment:
I got this from spread src:
_MATCH_ - Run `grep -q -e` on stdin. Without match, print error including content.

I tested with 'gpg (GnuPG) 1.' and 'gpg (GnuPG) 2.' and expression works in both cases.